about.html

```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>African Education Summit 2023</title>
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <!-- Mobile Menu Open/Close Toggle Button -->
+        <img
+          id="mobile-menu-toggle-button"
+          class="nav-bar__mobile-menu-toggle-button"
+          aria-expanded="false"
+          src="assets/icons/hamburger-open.svg"
+          alt="Open hamburger menu button"
+        />
+        <div id="nav-bar-wrapper" class="nav-bar-wrapper" data-visible="false">
+          <!-- Top bar Navigation -->
+          <div class="top-nav">
+            <div class="container">
+              <ul class="container top-nav__links-wrapper">
+                <div class="top-nav__social-links">
+                  <li>
+                    <a href="#" class="top-nav__link">
+                      <img
+                        class="top-nav__social-link"
+                        src="assets/icons/socials/facebook.svg"
+                        alt="Facebook logo"
+                      />
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="top-nav__link">
+                      <img
+                        class="top-nav__social-link"
+                        src="assets/icons/socials/twitter.svg"
+                        alt="Twitter logo"
+                      />
+                    </a>
+                  </li>
+                </div>
+                <li><a href="#" class="top-nav__link">Languages</a></li>
+                <li><a href="#" class="top-nav__link">My Page</a></li>
+                <li><a href="#" class="top-nav__link">Logout</a></li>
+              </ul>
+            </div>
+          </div>
+          <!-- Main Navigation -->
+          <div class="main-nav">
+            <div class="container">
+              <div class="main-nav__wrapper">
+                <a class="main-nav__logo-wrapper" href="/">
+                  <img
+                    id="main-nav__logo"
+                    class="main-nav__logo"
+                    src="assets/icons/logo-lg-light.svg"
+                    alt="African Eduction Summit Logo"
+                  />
+                </a>
+                <ul class="main-nav__links-wrapper">
+                  <li>
+                    <a href="about.html" class="main-nav__link active">About</a>
```

Comment:
- [ ] In the About page there should be a way to navigate from this page to the home page, kindly make sure to implement this. You can change this link to the `Home` page link so users can navigate your website easily